src/CMakeLists.txt

```diff
@@ -69,17 +69,38 @@ if ("${JUPYTER}" STREQUAL "ON")
    add_definitions(-DRB_XCODE)
 endif()
 
+#MESSAGE("Boost location hints:")
+#if (BOOST_ROOT)
+#  MESSAGE("  BOOST_ROOT: ${BOOST_ROOT}")
+#else()
+#  MESSAGE("  BOOST_ROOT not set")
+#endif()
+#if (BOOST_INCLUDEDIR)
+#  MESSAGE("  BOOST_INCLUDEDIR: ${BOOST_INCLUDEDIR}")
+#endif()
+#if (BOOST_LIBRARYDIR)
+#  MESSAGE("  BOOST_LIBRARYDIR: ${BOOST_LIBRARYDIR}")
+#endif()
 MESSAGE("Boost location hints:")
+if (BOOST_ROOT OR BOOST_INCLUDEDIR OR BOOST_LIBRARYDIR) 
+  set(Boost_NO_SYSTEM_PATHS ON)
+  set(Boost_NO_BOOST_CMAKE ON)
+  set(Boost_NO_WARN_NEW_VERSION ON)
```

Comment:
I presume that these warnings exist only to let users know that there might be new BOOST libraries available in this version of boost.  Since this isn't relevant to us, I can see turning these warnings off...

Reply:
Yeah, this one is not necessary, but more of a convenience. When there are multiple versions of boost there can be an overwhelming number of warning messages (a few lines per boost component).

Reply:
Maybe add an comment line like this?
# Suppress excessive warnings -- they aren't relevant for us